psutil/_psosx.py

```diff
@@ -230,6 +257,19 @@ def sensors_battery():
     return _common.sbattery(percent, secsleft, power_plugged)
 
 
+def sensors_fans():
+    """Return fans speed information.
+    """
+    ret = collections.defaultdict(list)
+    for key, value in cext.sensors_fans().items():
+        try:
+            ret[key].append(_common.sfan(key, int(value)))
+        except (OSError, IOError) as err:
```

Comment:
It doesn't look like this line can raise OSError or IOError

Reply:
Right, will remove